conf/etc/iem_ha.json

```diff
@@ -0,0 +1,42 @@
+{
+    "nodes": {
+        "any_host": {
```

Comment:
As there are some conflict and patch is old we can create new patch to add iem in HA.